src/components/icons/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './Alarm';
+export * from './Chevron';
+export * from './ChevronDownIcon';
+export * from './CrossIcon';
+export * from './DotsIcon';
+export * from './GearIcon';
+export * from './Info';
+export * from './PreviewCloseIcon';
+export * from './QuestionMarkIcon';
+export * from './Success';
+
+export type UIKitIcon = (props: React.SVGProps<SVGSVGElement>) => JSX.Element;
```

Comment:
Do we actually need this type? Why not `typeof Icon` for cases where it needed?

Reply:
It don\`t work because all of icons (in `/icons`) are `data` for `Icon` component, but not the `Icon` itself